syntaxes/test/cases.json

```diff
@@ -11,6 +11,12 @@
     "grammarFiles": ["syntaxes/inline-styles.json"],
     "testFile": "syntaxes/test/data/inline-styles.ts"
   },
+  {
+    "name": "expression syntax",
+    "scopeName": "expression.ng",
+    "grammarFiles": ["syntaxes/expression.json"],
+    "testFile": "syntaxes/test/data/expression.html"
+  },
   {
     "name": "template syntax",
```

Comment:
Let's load the expression syntax on this test case as well.

Reply:
I'm sorry, I don't think I understand what you mean.

Reply:
Could you add the expression grammar to the grammarFiles for the template syntax test case as well?

Reply:
Or attach to it a dummy grammar for expression.ng. If you look at the template snap file it looks like it’s not picking up the expression grammar at all.

Reply:
> @ghaschel could I help you with committing some of these changes? It could speed up the time of the PR.

I'm sorry for the (very) late response... I'm having the craziest of the weeks at work... I think tomorrow I'll be able to work on this... But any help would be appreciated

Reply:
No worries, man. I totally understand. Sounds good.

Reply:
I have added you as a collaborator